src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public int getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+        System.out.println(color.getRandomColor());
+        Random random = new Random();
+        int number = random.nextInt(100);
```

Comment:
Magic numbers should be avoided. The number 100 should be extracted to a constant with a meaningful name, as per the checklist.